### PR TITLE
Remove use of Array.prototype.includes to support older runtimes

### PR DESCRIPTION
**Pull request by PepperTeasdale**
Also removes `babel-polyfill` in favor of `transform-runtime` to surface these runtime-specific issues in testing.

@stubailo 👀  please

TODO:

- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests pass
- [x] Update CHANGELOG.md with your change
- [x] If this was a change that affects the external API, update the README

;)

### CODE REVIEW

**Comment by stubailo** [comment]
(BTW - you can use `[x]` to put checked boxes :])

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ### vNEXT
 
+### v0.8.3
```
Comment (stubailo) [inline CHANGELOG.md]: For future PRs, please don't set the version number here and in package.json - sometimes we publish several things in one version

File: test/index.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 // This file cannot be written with ECMAScript 2015 because it has to load
 // the Babel require hook to enable ECMAScript 2015 features!
 require('babel-core/register');
-require('babel-polyfill');
+require("babel-core").transform("code", {
+  plugins: ["transform-runtime"]
```
Comment (stubailo) [inline test/index.js]: What does this change do?

Comment (stubailo) [inline test/index.js]: Oh never mind just read the description :]

Comment (PepperTeasdale) [inline test/index.js]: If there's something I can do to clarify the intention of this in the code, I'm super happy to add it.

**Comment by PepperTeasdale** [comment]
@stubailo Thanks for the feedback :haircut_man:

**Comment by stubailo** [comment]
LGTM, @jnwng I think it's up to you to merge!

**Comment by jnwng** [comment]
thank you @PepperTeasdale!

**Pull request merged**